models/object_detection/model_templates/horizontal-text-detection/readme.md

```diff
@@ -22,165 +27,99 @@ Activate virtual environment:
+   -m export/model.xml \
+   -at ssd \
+   -i /dev/video0
 ```
 
-### 6. Training and Fine-tuning
-
-Try both following variants and select the best one:
-
-   * **Training** from scratch or pre-trained weights. Only if you have a lot of data, let's say tens of thousands or even more images. This variant assumes long training process starting from big values of learning rate and eventually decreasing it according to a training schedule.
-   * **Fine-tuning** from pre-trained weights. If the dataset is not big enough, then the model tends to overfit quickly, forgetting about the data that was used for pre-training and reducing the generalization ability of the final model. Hence, small starting learning rate and short training schedule are recommended.
-
-   * If you would like to start **training** from pre-trained weights use `--load-weights` pararmeter.
-
-      ```bash
-      python train.py \
-         --load-weights ${WORK_DIR}/snapshot.pth \
-         --train-ann-files ${TRAIN_ANN_FILE} \
-         --train-data-roots ${TRAIN_IMG_ROOT} \
-         --val-ann-files ${VAL_ANN_FILE} \
-         --val-data-roots ${VAL_IMG_ROOT} \
-         --save-checkpoints-to ${WORK_DIR}/outputs
-      ```
-
-      Also you can use parameters such as `--epochs`, `--batch-size`, `--gpu-num`, `--base-learning-rate`, otherwise default values will be loaded from `${MODEL_TEMPLATE}`.
-
-   * If you would like to start **fine-tuning** from pre-trained weights use `--resume-from` parameter and value of `--epochs` have to exceed the value stored inside `${MODEL_TEMPLATE}` file, otherwise training will be ended immediately. Here we add `5` additional epochs.
+### 4. Fine-tune
 
-      ```bash
-      export ADD_EPOCHS=5
-      export EPOCHS_NUM=$((`cat ${MODEL_TEMPLATE} | grep epochs | tr -dc '0-9'` + ${ADD_EPOCHS}))
-
-      python train.py \
-         --resume-from ${WORK_DIR}/snapshot.pth \
-         --train-ann-files ${TRAIN_ANN_FILE} \
-         --train-data-roots ${TRAIN_IMG_ROOT} \
-         --val-ann-files ${VAL_ANN_FILE} \
-         --val-data-roots ${VAL_IMG_ROOT} \
-         --save-checkpoints-to ${WORK_DIR}/outputs \
-         --epochs ${EPOCHS_NUM}
-      ```
+#### a. Prepare dataset
 
-### 7. Evaluation
+In this toy example we use same images as training, validation and test subsets, but we strictly recommend not to use the same data for training, validation and test. This particular example is for demonstration of model quality growth on particular dataset during fine-tuning only. See more about dataset split [here](https://en.wikipedia.org/wiki/Training,_validation,_and_test_sets). In order to train a model that would be quite similar in terms of quality to exising pre-trained model one can use this [section](datasets.md) to prepare publicly-available datasets for training. One can also use its own preliminary annotated dataset. Annotation can be created using [CVAT](https://github.com/openvinotoolkit/cvat) as we did in this toy example.
```

Comment:
Also, I suggest you start "In order to train a model" as a new paragraph to separate the "toy" training and the public one. Also, it'll allow seeing a link with datasets.md more clearly, because now it is hiding in the text.

Reply:
fixed